src/main/java/p76/bicycles/dto/Bicycle.java

```diff
@@ -15,10 +14,12 @@
     private RearDerailleur rearDerailleur;
     private Brakes brakes;
     private Wheels wheels;
+    private static int ID = 1;
```

Comment:
why static? One common ID for all bicycles?

Reply:
Static as i don't won't to change initial value. Every bicycle has ID ++ and that generates correctly.

Reply:
STATIC means that it belongs to Class not to the objects. Every Bicycle has the same ID. It is wrong